apps/contract-sync-job/src/utils.ts

```diff
@@ -0,0 +1,31 @@
+import fetch from 'node-fetch';
```

Comment:
Might be best to rename this file to ipfs? and  remove the logger since it isn't being used